packages/manager/apps/adp/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
Be careful, if you edit package(s) with a `feat` commits, all packages updated by this commit will be bumped in the future release.
Prefer create another commit (`docs` for example) specific for markdown update.

Reply:
Sure. Point noted :)